--- setup.py.orig	2022-11-22 13:11:03.752251443 +1000
+++ setup.py	2022-11-22 13:11:16.648100418 +1000
@@ -28,7 +28,7 @@
 # Are we installing the command line scripts?
 # this is an experimental option for users who are
 # using the Python entry point feature of setuptools and Conda instead
-NO_INSTALL_CMDLINE = int(os.getenv('RIOS_NOCMDLINE', '0')) > 0
+NO_INSTALL_CMDLINE = True
 
 if NO_INSTALL_CMDLINE:
     # still install the scripts for the parallel processing to work properly
